chips/README.md

```diff
@@ -0,0 +1,302 @@
+# Chip components
+
+Chips appear dynamically as a group of multiple interactive elements. With chips, users can filter
+content, enter information, make selections or trigger actions.
+
+## Chip design specs
+
+You can find the design specs on [decathlon.design](https://www.decathlon.design/).
+
+## Usage
+
+If you want to use components of this module in your android mobile application, you should
+first add the Gradle dependency in your Gradle file:
+
+```kotlin
+implementation("com.decathlon.vitamin.compose:chips:<versions>")
+```
+
+### Action
+
+Action chip offers actions related to the content.
+It appears dynamically and contextually in the interface.
+
+```kotlin
+object VitaminChips {
+    
+    @Composable
+    fun Action(
+        label: String,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        colors: VitaminChipColors = VitaminChipDefaults.default(),
+        sizes: ChipSizes = VitaminChipSizes.medium(),
+        ripple: RippleTheme = VitaminTheme.ripples.brand,
+        onClick: () -> Unit,
+        leadingIcon: (@Composable () -> Unit)? = null
+    )
+}
+```
+
+Parameters | Descriptions
+-- | --
+`label: String` | Text to be displayed into the Chip
+`modifier: Modifier = Modifier` | `Modifier` to be applied to the component
+`enabled: Boolean = true` | True if you can click on the Chip, otherwise false
+`colors: ChipColors = VitaminChipColors.default()` | Colors to be applied to the Chip
+`sizes: ChipSizes = VitaminChipSizes.medium()` | Sizes to be applied to the Chip. (VitaminChipSizes.medium() | VitaminChipSizes.small())
+`ripples: RippleTheme` | The ripple effect to be applied to the Chip
+`onClick: (() -> Unit)?` | Callback to be called when the action Chip is clicked
+`leadingIcon: (@Composable () -> Unit)? = null` | Optional Content to be displayed before label into the Chip
+
+_In your project :_
+```kotlin
+VitaminChips.Action(
+    label = "Share",
+    onClick = {},
+    leadingIcon = {
+        Icon(
+            painter = painterResource(id = R.drawable.ic_vtmn_share_line),
+            contentDescription = null
+        )
+    }
+)
+```
+
+### Filter Chip
+
+Filter chips are used as filters for list or content.
+Multiple chips can be selected or unselected.
+A selected Filter Chip displays a tick as start content.
+
+```kotlin
+object VitaminChips {
+
+    @Composable
+    fun Filter(
+        label: String,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        selected: Boolean = false,
+        colors: VitaminSelectableChipColors = VitaminChipDefaults.selectable(),
+        sizes: ChipSizes = VitaminChipSizes.medium(),
+        ripple: RippleTheme = VitaminTheme.ripples.brand,
+        onClick: () -> Unit,
+        selectedIcon: (@Composable () -> Unit) = { VitaminChipButtons.Selected() }
```

Comment:
You should scope the type with `VitaminChipButtons`, no?
```
selectedIcon: (@Composable VitaminChipButtons.() -> Unit) = { VitaminChipButtons.Selected() }
```

Reply:
Plus, why the default value is `Selected` if the default value of `selected` parameter is `false`? 🤔

Reply:
For the scope, same comment for other components

Reply:
selectedIcon is configurable if you want to change the icon when chip is selected. But Maybe I could just remove this parameter. The tick is the icon Vitamin chose

Reply:
I removed this input because we don't want it customizable for now